src/Nethermind/Nethermind.Runner/Properties/launchSettings.json

```diff
@@ -267,9 +267,9 @@
       "commandName": "Docker",
       "commandLineArgs": "-c goerli -dd /data --JsonRpc.Enabled true --JsonRpc.Host 0.0.0.0"
     },
-    "Denver": {
+    "Chiado": {
       "commandName": "Project",
-      "commandLineArgs": "-c denver -dd %NETHERMIND_DATA_DIR%",
+      "commandLineArgs": "-c chiado -dd %NETHERMIND_DATA_DIR%  --Init.DiagnosticMode MemDb",
```

Comment:
delete DiagnosticMode?

Reply:
why? we have lots of networks as MemDb in launchsettings. It has sense IMO, because you won't have garbage in your dev disk.